src/_data/sites/leumi.yml

```diff
@@ -0,0 +1,8 @@
+name: Bank Leumi (Israel)
+description: |-
+  - Password consists of 6 to 12 characters
+  - Password contains only english letters and numbers without spaces.
+  - הסיסמה מורכבת מ-6 עד 12 תווים ומכילה אותיות באנגלית וספרות
```

Comment:
I don’t think we need this line since the same is already noted in English above.

Reply:
I removed the duplicate line in e5a3437